src/index.tsx

```diff
@@ -65,6 +74,25 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
     sdkVersion,
   });
 
+  // by default use non-expiring assignment cache.
+  // this is the recommended setting for most clients.
+  const assignmentCacheBehavior =
+    config.assignmentCacheBehavior || DEFAULT_ASSIGNMENT_CACHE_BEHAVIOR;
+  switch (assignmentCacheBehavior) {
+    case 'disabled':
+      EppoReactNativeClient.instance.disableAssignmentCache();
+      break;
+    case 'expiring':
+      const cacheSize =
+        config.assigmentExpiringCacheSize ||
+        DEFAULT_EXPIRING_ASSIGNMENT_CACHE_SIZE;
+      EppoReactNativeClient.instance.useLRUAssignmentCache(cacheSize);
+      break;
+    case 'non-expiring':
```

Comment:
should this also be our `default:` behavior? It's an edge case as we _shouldn't_ ever get there, but I do like defensive coding. What happens if nothing matches and we don't call any cache functions?

Reply:
yea thanks I had the same thought - wish this compiler enforced it